mm-mock-server/src/mocks/queries.ts

```diff
@@ -18,12 +18,17 @@ export function mockQueries(serverState: MockServerState) {
                 generators.generateChannel([serverState.loggedInUser, serverState.otherUsers[0]]),
             ]
         },
-        findUsers: () => {
+        findChannelInvitationsForUser: () => {
```

Comment:
I'm trying to decide whether it makes sense to have this field. 

The backend has a similar field, `findPendingChannelInvitationsForUser`, which returns the all channel invitation that the user has sent out or received and that haven't yet been accepted or declined. 

The backend does have `findChannelInvitationsForUser`, but it returns all invitations, including those that have already been accepted or declined. Not sure why the app would need that.

Even `findPendingChannelInvitationsForUser` may not be needed, as the pending channel invitations are listed in `UserInbox`. It should be possible to pull the user's inbox and retrieve from there everything the app needs to show a listing of new items, including invitations and new messages.